cmd/ddltest/ddl_test.go

```diff
@@ -304,7 +304,10 @@ func (s *TestDDLSuite) startServer(i int, fp *os.File) (*server, error) {
 		}
 		log.Warnf("ping addr %v failed, retry count %d err %v", addr, i, err)
 
-		db.Close()
+		err = db.Close()
+		if err != nil {
+			panic(err)
```

Comment:
I  think we can print the log and break it here.

Reply:
Done.